src/bundler.js

```diff
@@ -370,12 +381,27 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
 
                     } else {
 
-                        bundleStatsCollector.ParseJsForStats(jsBundle, code);
-                        next({
-                            code: code,
-                            path: jsPath,
-                            originalPath: filePath
-                        });
+                        if (options.webpack) {
+
+                            var cleanedCode = sourceMap.removeComments(code);
+                            cleanedCode = sourceMap.removeMapFileComments(cleanedCode);

```

Comment:
Since dealing with webpack source map comments is hairy, this just strips them out of the code before we continue.